sdcm/nemesis.py

```diff
@@ -2604,18 +2604,21 @@ def check_ssl_reload_log(node_system_log):
                                                     ignore_status=True).stdout
         update_certificates()
         node_system_logs = {}
-        for node in self.cluster.nodes:
-            node_system_logs[node] = node.follow_system_log(
-                patterns=f'messaging_service - Reloaded {{{ssl_files_location}}}')
-            node.remoter.send_files(src='data_dir/ssl_conf/db.crt', dst='/tmp')
-            node.remoter.run(f"sudo cp -f /tmp/db.crt {ssl_files_location}")
-            new_crt = node.remoter.run(f"cat {ssl_files_location}").stdout
-            if in_place_crt == new_crt:
-                raise Exception('The CRT file was not replaced')
+        with DbEventsFilter(
```

Comment:
Can you add a note, we need to remove the filter when the issue is fixed in scylla (https://github.com/scylladb/scylla/issues/7909#issuecomment-758062545)?

Reply:
done